pkg/utils/config-manager/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Build Stage
```

Comment:
These should all be in `cmd/config-manager` and not anywhere under `pkg` (it has a lot of non-golang cruft and the golang package is main).